README.md

```diff
@@ -137,6 +137,13 @@ double-precision topography file.
 Options
   * `--vgrid <vgrid>`  vertical grid (default 'ocean_vgrid.nc')
 
+## cut_off_T_cells
```

Comment:
I don't find "cut_off_T_cells" intuitive or self-explanatory. Why not "min_dy"? This would be more consistent with `min_max_depth`.

Reply:
Thanks Andrew, I changed it